docs/standard/serialization/system-text-json/migrate-from-newtonsoft.md

```diff
@@ -257,7 +257,7 @@ For more information, see [Preserve references and handle circular references](p
 
 ### Dictionary with non-string key
 
-Both `Newtonsoft.Json` and `System.Text.Json` support collections of type `Dictionary<TKey, TValue>`. However, in `System.Text.Json`, `TKey` must be a primitive type, not a custom type. For more information, see [Supported key types](supported-collection-types.md#supported-key-types).
+Both `Newtonsoft.Json` and `System.Text.Json` support collections of type `Dictionary<TKey, TValue>`. However, in `System.Text.Json`, `TKey` must be a primitive type, not a custom type. For more information, see [Supported key types](supported-types.md#supported-key-types).
```

Comment:
This paragraph is out of date since the introduction of the `JsonConverter.WriteAsPropertyName`/`ReadAsPropertyName` methods which lets users add dictionary key support for any type they wish. When it comes to built-in support, it also includes all the supported non-primitive types that serialize as strings including `DateTimeOffset`, `TimeSpan`, `Uri`, and `Version`.

Reply:
I see this is already documented in the "supported key types" section in the main document, so I think we can just defer to that.